src/types.ts

```diff
@@ -464,3 +467,58 @@ export type IEmailGetResponse = IGetResponse<IEmailProperties>;
 export type IEmailSetArguments = ISetArguments<IEmailProperties>;
 
 export type IEmailSetResponse = ISetResponse<IEmailProperties>;
+
+/**
+ * See https://jmap.io/spec-mail.html#email-submission
+ */
+export interface DeliveryStatus {
+  smtpReply: string;
+  delivered: 'queued' | 'yes' | 'no' | 'unknown';
+  displayed: 'unknown' | 'yes';
+}
+
+/**
+ * See https://jmap.io/spec-mail.html#email-submission
+ */
+export interface Address {
+  email: string;
+  parameters: any | null;
+}
+/**
+ * See https://jmap.io/spec-mail.html#email-submission
+ */
+export interface Envelope {
+  mailFrom: Address;
+  rcptTo: Address[];
+}
+/**
+ * See https://jmap.io/spec-mail.html#email-submission
+ */
+export interface IEmailSubmissionProperties {
+  id: string;
+  identityId: string;
+  emailId: string;
+  threadId: string;
+  envelope: Envelope | null;
+  sendAt: Date;
+  undoStatus: 'pending' | 'final' | 'canceled';
+  deliveryStatus: { [id: string]: DeliveryStatus } | null;
+  dsnBlobIds: string[];
+  mdnBlobIds: string[];
+}
+
+/**
+ * See https://jmap.io/spec-mail.html#emailget
```

Comment:
Why do you link https://jmap.io/spec-mail.html#emailget in somethig trealted to `EmailSubmission/get`?

https://jmap.io/spec-mail.html#emailsubmissionget maybe?